src/scss/includes/direction.scss

```diff
@@ -817,18 +819,18 @@ input:valid:focus ~ .itinerary__field__clear {
     }
   }
 
-  .direction_shortcut {
+  button.direction_shortcut {
     display: block;
     width: 48px;
     height: 48px;
-    background: #fff;
     border-radius: 50%;
     box-shadow: rgba(0,0,0,.2) 0 2px 12px 0;
     position: fixed;
     bottom: 66px;
     right: 9px;
-    color: #5c6f84;
     font-size: 25px;
+    opacity: 1;
+    transition: opacity .3s;
```

Comment:
You used 0.2s as duration on the other button. If there is no reason for this difference it should be the same value.